src/main/scala/ru/org/codingteam/keter/game/objects/EventQueue.scala

```diff
@@ -29,5 +29,7 @@ sealed class EventQueue(val timestamp: Double, events: SortedMap[Double, Seq[Sch
 }
 
 object EventQueue {
+  import Ordering.Double.IeeeOrdering
```

Comment:
Here we're required to add this implicit, since `SortedMap` requires that.

We're now required to add these implicits in very random places of our code which is bad.

I think we better declare our own type for time (wrap `Double` or think better about that), but, for now, this will work. Let's create a new task about this refactoring, and do it after merge?

@ttldtor, your thoughts?

Reply:
@ForNeVeR sounds good. A created the #91